src/main/perl/RuleBasedEditor.pm

```diff
@@ -532,12 +573,36 @@ sub _formatConfigLine
 
 =pod
 
+=item _escape_regexp_string
+
+Help method to escape all characters with a special interpretation in the context
+of a regexp.
+
+Arguments:
+    regexp_str: initial regexp string (characters not escaped)
+
+Return value:
+    string: regexp with all specail characters escaped
+
+=cut
+
+sub _escape_regexp_string {
```

Comment:
would also be nice that this metohd itself is unittested, not only when called from within `_buildLinePattern`